includes/api/ApiQueryUserContributions.php

```diff
@@ -486,4 +500,26 @@ public function getHelpUrls() {
 	public function getVersion() {
 		return __CLASS__ . ': $Id$';
 	}
+
+	private function normalizeValues( $values ) {
+		return array_map( function ($value) {
+			if (is_numeric($value)) {
+				return $value;
+			}
+
+			return $this->getDB()->addQuotes($value);
+		}, $values );
+	}
+
+	private function createWhereCondition( $name, $values ): string {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "createWhereCondition" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1144)